### ISSUE TITLE
Cellpose results better online

### ISSUE THREAD

**Issue by Daria150393** [MouseLand/cellpose#547]
Hi,

when I try out cellpose [online](https://www.cellpose.org/), I get better results

![online results](https://user-images.githubusercontent.com/44135481/183643647-a5327fc1-5e5f-4467-af2b-2cbb5e319dcc.png) 

than when I run the segmentation in the [jupyter notebook](https://github.com/MouseLand/cellpose/blob/main/notebooks/run_cellpose.ipynb) with cellpose 1.0

![1.0](https://user-images.githubusercontent.com/44135481/183644690-d3c212f7-0588-4a07-8e90-ec17fa7a74da.png) 

or 
2.0.:

![2.0](https://user-images.githubusercontent.com/44135481/183644933-ecad8c3a-339b-4bb9-82e6-51643a6cf4b2.png)

Where do these differences in performance come from? I used the very same image
![image](https://user-images.githubusercontent.com/44135481/183645946-193b8243-41ec-4a22-9ac6-e9d74996afb0.jpg)

and settings both online:

-What to segment: nuclei
-channel to segment: gray
-nuclear channel: none
-avg cell diameter: 7

and in the notebook:

img = cv2.resize(img_full, dsize=(512,375))
model = models.Cellpose(gpu=False, model_type='nuclei')
channels = [0,0]
masks, flows, styles, diams = model.eval(img, diameter=7, channels=channels)

Is some kind of preprocessing done on the image in the online version? 

Do you have any other advice on how to improve the segmentation of these neutrophils?

Thanks a lot for all your effort, great work!

**Comment by carsen-stringer** [comment]
the diameter online is a minimum of 7.5; also online the mxnet models are used not the pytorch models, so that will cause differences.

to my eye, all the outputs look similar, you can find more cells by turning off the quality control step `model.eval(img, flow_threshold=0, ...)`

also sometimes the `'cyto'` model works better than the nuclei model, particularly if the nuclei are inhomogeneous

going to close this issue but let us know if there are still issues

**Comment by Daria150393** [comment]
Thank you so much for your advice! The cyto model really performs better on the segmenation of those c-shaped nuclei and we are also able to detect more cells! This helped us a lot.

What we are still struggling with is that 

1. The neutrophils are often split up into two cells as can be seen in the image below. Is there some "fill-holes" post-processing parameter that could fix this? Or could the flows be used to improve the segmentation, as it looks like they are correctly merging such cells. 

Raw image:
<img width="711" alt="Bildschirmfoto 2022-08-19 um 14 49 25" src="https://user-images.githubusercontent.com/44135481/185622637-7c1aaec0-72da-4bbf-9f84-5fae168bdad4.png">

Corresponding mask:
<img width="717" alt="Bildschirmfoto 2022-08-19 um 14 48 47" src="https://user-images.githubusercontent.com/44135481/185622707-ff8b2ee8-1be3-4158-aca4-6509238a48df.png">

Corresponding flows:
<img width="616" alt="Bildschirmfoto 2022-08-19 um 14 51 24" src="https://user-images.githubusercontent.com/44135481/185622746-607aa96a-0afd-4364-8a9b-5c9a56af46ed.png">

2. Cells with a donut-shaped nucleus don't have holes in their masks. Again our question is whether this can be changed based on some post-processing parameters. Otherwise we would try to improve this by threshold-based post-processing of the masks outside of cellpose.

Raw image:
<img width="532" alt="Bildschirmfoto 2022-08-19 um 14 57 28" src="https://user-images.githubusercontent.com/44135481/185623401-ae1370d6-e9bf-4d55-a527-0c5fcb492b9a.png">

Corresponding mask:
<img width="542" alt="Bildschirmfoto 2022-08-19 um 14 56 33" src="https://user-images.githubusercontent.com/44135481/185623433-6b7c1d2e-572d-417f-9657-fe4475adf907.png">

Thank you for your help!
